src/components/Filter/Filter.react.js

```diff
@@ -61,7 +61,8 @@ let Filter = ({ schema, filters, renderRow, onChange, blacklist, className }) =>
         }
 
         // Get the column preference of the current class.
-        const currentColumnPreference = currentApp.columnPreference[className];
+        const currentColumnPreference = currentApp.columnPreference ? currentApp.columnPreference[className]
+        : null;
```

Comment:
Is this linted? I'd assume `: null` either goes into the line above or would be indented.

Reply:
linting did not work for me. Fixed it manually.